roles/nighthawk/templates/workload.yml.j2

```diff
@@ -0,0 +1,135 @@
+          - name: es_index
+            value: "{{ elasticsearch.index_name | default("ripsaw-nighthawk") }}"
+          - name: parallel
+            value: "{{ elasticsearch.parallel | default(false) }}"
+          - name: es_verify_cert
+            value: "{{ elasticsearch.verify_cert | default(true) }}"
+{% endif %}
+{% if prometheus is defined %}
+          - name: prom_es
+            value: "{{ prometheus.es_url }}"
+          - name: prom_parallel
+            value: "{{ prometheus.es_parallel | default(false) }}"
+          - name: prom_token
+            value: "{{ prometheus.prom_token | default() }}"
+          - name: prom_url
+            value: "{{ prometheus.prom_url | default() }}"
+{% endif %}
+          - name: client_node
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          - name: server_node
+            value: "{{ workload_args.pin_server|default("unknown") }}"
+{% if workload_args.client_resources is defined %}
+        resources: {{ workload_args.client_resources | to_json }}
+{% endif %}
+        imagePullPolicy: Always
+        workingDir: /tmp
+        command: ["/bin/bash", "-c"]
+        args:
+          - |
+{% if item.metadata.labels.termination|default("http") == "http" %}
+             protocol="http";
+{% else %}
+             protocol="https";
+{% endif %}
+{% if workload_args.url is defined %}
+             url={{ workload_args.url }}
+{% else %}
+             url=$protocol://{{ item.metadata.labels.route }}-{{ operator_namespace }}.{{ routerCanonicalName.split('.')[1:] | join('.') }}
+{% endif %}
+             exit_status=0;
+             while true; do
+               if [[ $(redis-cli -h {{bo.resources[0].status.podIP}} get start-{{ trunc_uuid }}) =~ 'true' ]]; then
+                 run_snafu -t nighthawk -s {{workload_args.samples|int}} --resourcetype {{ resource_kind }} -u {{ uuid }} --user {{test_user | default("ripsaw")}} --url $url --concurrency {{ workload_args.concurrency|int }} --duration {{ workload_args.duration|int }} --connections {{ workload_args.connections|int }} --max-requests-per-connection {{ workload_args.max_requests_per_connection|int }} --create-archive --archive-file /tmp/nighthawk.archive \
+{% if workload_args.run_id is defined %}
+                 --run-id {{workload_args.run_id}} \
+{% endif %}
+{% if workload_args.debug is defined and workload_args.debug %}
+                 -v \
+{% endif %}
+                 ;
+                      if [[ $? -ne 0 ]]; then
+                         exit_status=1;
+                      fi;
+               else
+                 continue;
+               fi;
+               break;
+             done;
+             redis-cli -h {{bo.resources[0].status.podIP}} set start-{{ trunc_uuid }} false;
+             exit $exit_status;
+      restartPolicy: Never
+{% if workload_args.pin is sameas true %}
+      nodeSelector:
+          kubernetes.io/hostname: '{{ workload_args.pin_client }}'
+{% endif %}
```

Comment:
Should include the metadata-collection include statement like https://github.com/cloud-bulldozer/benchmark-operator/blob/0e6043a1473c895ea381712effa35e4c27808850/roles/uperf/templates/server.yml.j2#L70